wp-content/themes/twentyseventeen-wp20/style.css

```diff
@@ -893,7 +893,7 @@ body.page-slug-whats-on #primary .entry-header {
 }
 
 body.page-slug-whats-on h1.entry-title {
-	width: 639px;
+	flex-basis: 55%;
 	font-size: clamp( 20px, 2.4vw, 30px);
 	margin-right: var(--wp20--spacing--medium);
```

Comment:
Can we remove this on desktop? It'll make the layout match more closely to the design.

Reply:
Done in 8e20295 👍🏻